src/actix/api/service_api.rs

```diff
@@ -106,8 +108,16 @@ async fn livez() -> impl Responder {
 }
 
 #[get("/readyz")]
-async fn readyz() -> impl Responder {
-    kubernetes_healthz().await
+async fn readyz(ready: web::Data<Ready>) -> impl Responder {
+    let (status, body) = if ready.ready().await {
+        (StatusCode::OK, "all shards are ready")
+    } else {
+        (StatusCode::SERVICE_UNAVAILABLE, "some shards are not ready")
+    };
+
+    HttpResponse::build(status)
+        .content_type(ContentType::plaintext())
+        .body(body)
```

Comment:
I think we should format this differently, better matching https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health

We probably want something like:

```
[+]shards/initial-ready ok
healthz check passes
```

```
[-]shards/initial-ready fail
healthz check failed
```

though the fine details of it are unclear to me.

Reply:
_As far as I understand_, that's how _Kubernetes_ will format its output based solely on health endpoints status code. We don't need to format response body (or provide any) in any way.

Reply:
Looks like you're right! Resources on it are hard to find.

In that case shouldn't we just return HTTP 200 with `ready`